test/MUXControlsReleaseTest/MUXControls.ReleaseTest/NugetTestsCX.cs

```diff
@@ -59,5 +59,16 @@ public void RepeaterNoCrashTest()
             var item3 = FindElement.ByName("Item3");
             Verify.IsNotNull(item3);
         }
+
+        [TestMethod]
+        public void CornerRadiusTest()
+        {
+            var button = new Button(FindElement.ByName("GetCheckBoxRectangleCornerRadiusValue"));
+            button.Click();
```

Comment:
Recommend using Invoke() instead of Click() (which is mouse input) for better speed and reliability